server/controllers/authControllers.js

```diff
@@ -109,10 +111,14 @@ authControllers.createUsers = (req, res, next) => {
               trainer_id,
             ];
             db.query(
-              `INSERT INTO clients (username, password, first_name, last_name, age, weight, height, gender, trainer_id) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9);`,
+              `INSERT INTO clients (username, password, first_name, last_name, age, weight, height, gender, trainer_id) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9) RETURNING client_id;`,
               values
             )
-              .then((data) => next())
+              .then((data) => {
+                res.locals.userType = userType;
```

Comment:
Great job setting up the middlewares. I've noticed that you've been using promises throughout your middleware, I think for the sake of readability we should start implementing async /await instead. Let me know if this works for you.

Reply:
Hi Dennis, 
Thank you for your constructive feedback. I agree that async/await syntax does look cleaner and achieve the same purpose. I could try and implementing it next time.